src/main/java/com/code_intelligence/jazzer/agent/RuntimeInstrumentor.kt

```diff
@@ -57,6 +57,17 @@ class RuntimeInstrumentor(
         protectionDomain: ProtectionDomain?,
         classfileBuffer: ByteArray,
     ): ByteArray? {
+        var pathPrefix = ""
+        if (Opt.instrumentOnly && protectionDomain != null) {
```

Comment:
`protectionDomain` was already null-checked so that `?` isn't required anymore. Also the methods calls on `protectionDomain` could be changed to  a property access for more idiomatic Kotlin code.